src/features/pages/home/live-pricing/table-component/live-market-table.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react'
+import {
+    flexRender,
+    getCoreRowModel,
+    getSortedRowModel,
+    useReactTable,
+} from '@tanstack/react-table'
+import type { SortingState } from '@tanstack/react-table'
+import { TAvailableLiveMarkets, TMarketData } from '../types'
+import useLiveColumns from '../use-live-columns'
+import { table_row_header, table_row_data } from './live-pricing.module.scss'
+import Flex from 'features/components/atoms/flex-box'
+import useRegion from 'components/hooks/use-region'
+import Link from 'features/components/atoms/link'
+import { Localize } from 'components/localization'
+
+export type TLiveMarketTableProps = {
+    selected_market: TAvailableLiveMarkets
+    link_to: string
+}
+
+const LiveMarketTable = ({ selected_market, link_to }: TLiveMarketTableProps) => {
+    const { is_eu } = useRegion()
+    const [initial_loaded, setInitialLoaded] = useState(false)
+
+    const [rawMarketsData, setRawMarketsData] = useState()
+    const TABLE_VISIBLE_ROWS = 5
+    const [markets_data, setMarketsData] = useState(() => {
+        const temp = new Map<TAvailableLiveMarkets, TMarketData[]>()
+        return temp
+    })
+    const intervalRef = useRef(null)
+
+    const [sorting, setSorting] = React.useState<SortingState>([])
+
+    const region = useMemo(() => {
+        return is_eu
+            ? 'https://deriv-static-pricingfeed.firebaseio.com/eu.json'
+            : 'https://deriv-static-pricingfeed.firebaseio.com/row.json'
+    }, [is_eu])
```

Comment:
no need to use memo, because we check it only once
to be sure that it happen once, better to use useEffect with empty array